decks/contributing-to-ansible.html

```diff
@@ -0,0 +1,517 @@
+      <div class="slides">
+        <section data-state="cover">
+          <p class="ans-logo"><img src="images/ansible-wordmark-white.svg" width="260" alt="" /></p>
+          <h1>Contributing to Ansible</h1>
+          <p>docs.ansible.com/ansible/devel/community</p>
+          <aside class="notes">
+              <ul>
+                  <li>Welcome, thank you for taking the time to join us today</li>
+                  <li>Presentation is being recorded, you'll receive an email with detail</li>
+                  <li>docs.ansible.com/ansible/community</li>
+                  <li></li>
+                  <li></li>
+              </ul>
+          </aside>
+         </section>
+        <section data-state="cover">
+            <h2>Who am I?</h2>
+            <code><pre class="language-yaml">
+  - name: John Barker
+    aliases: gundalow
+    title: Principal Software Engineer
+    org: Ansible, by Red Hat
+    start_date: 2016
+    roles:
+      - community
+      - testing
+    freenode: gundalow
+    github: gundalow
+    twitter: the_gundalow
+    email: gundalow@redhat.com
+          </pre></code>
+          <aside class="notes">
+              <ul>
+                <li>That's me, what about you</li>
+                <li>SHOW OF HANDS: Who's just starting out with Ansible</li>
+                <li>SHOW OF HANDS: Who has a complex system that they want to simplify</li>
+                <li>SHOW OF HANDS: Who's got more they want to automate with Ansible</li>
+                <li>SHOW OF HANDS: Who's a contributor - raised PRs (modules, bugs, docs) - Thank you</li>
+                <li>SHOW OF HANDS: Who considers themselves an expert? - Excellent, you can answer the difficult Q&A at the end</li>
+              </ul>
+
+          </aside>
+        </section>
+
+        <section class="text-center" data-state="cover">
+          <svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" width="300" viewBox="-449 450 125 125" style="enable-background:new -449 450 125 125;" xml:space="preserve">
+            <g id="XMLID_3_">
+              <circle id="XMLID_7_" class="circle" cx="-386.5" cy="512.5" r="62"></circle>
+              <path id="XMLID_4_" class="a-mark" d="M-356.9,537.1l-24.7-59.4c-0.7-1.7-2.1-2.6-3.9-2.6c-1.7,0-3.2,0.9-4,2.6l-27.1,65.2h9.2 l10.7-26.9l32,25.9c1.3,1,2.2,1.5,3.4,1.5c2.4,0,4.6-1.8,4.6-4.5C-356.5,538.5-356.6,537.8-356.9,537.1z M-385.4,488.4l16.1,39.6 l-24.2-19L-385.4,488.4z"></path>
+            </g>
+          </svg>
+          <h2>THE ANSIBLE COMMUNITY</h2>
+          <p>The humans that get involved, even once, make us stronger</p>
+          <p>We will cover</p>
+          <ol>
+              <li>Scale and skill set - what <i>you</i> can bring</li>
+              <li>How to find things to help with</li>
+              <li>PR workflow</li>
+              <li>How to review PRs</li>
+              <li>How to join the conversation and shame the future</li>
```

Comment:
change "shame" to "shape" on this line

Reply:
I feel like shame really drives the point home though ;) heh

Reply:
woops, the shame, the shame